__tests__/app.test.js

```diff
@@ -107,3 +107,40 @@ describe('/api/articles/:article_id', () => {
 		});
 	});
 });
+
+describe('/api/articles', () => {
+	describe('200: GET:', () => {
+		it('responds with status 200 and an array of object with the correct shape', () => {
+			return request(app)
+				.get('/api/articles')
+				.expect(200)
+				.then(({ body: { articles } }) => {
+					expect(articles).toBeInstanceOf(Array);
+					expect(articles).toHaveLength(12);
+					articles.forEach(article => {
+						expect(article).toMatchObject({
+							article_id: expect.any(Number),
+							title: expect.any(String),
+							topic: expect.any(String),
+							author: expect.any(String),
+							body: expect.any(String),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							article_img_url: expect.any(String),
+						});
```

Comment:
This is a very good test! because you're using a forEach you could drop the assertion `toBeInstanceOf(Array)` if you wanted